### PR TITLE
Ran PHPCS on htdocs directory

### PR DESCRIPTION
**Pull request by driusan**
I've fixed all the phpcs errors in htdocs and added it to Travis, to ensure that the coding style stays clean.

### CODE REVIEW

**Comment by driusan** [comment]
@mohadesz can you test this branch? In particular, the feedback_bvl and feedback_mri popups, and request new accounts are the most likely thing to have broken..

File: htdocs/feedback_bvl_popup.php [hunk 1]
```diff
@@ -1,10 +1,18 @@
 <?php
-set_include_path(get_include_path().":../project/libraries:../php/libraries:");
-ini_set('default_charset', 'utf-8');
```
Comment (mohadesz) [inline htdocs/feedback_bvl_popup.php]: include this :
require_once __DIR__ . "/../vendor/autoload.php";

File: htdocs/feedback_mri_popup.php [hunk 2]
```diff
@@ -1,142 +1,173 @@
-<?php
-/**
- * @package mri
- */
-set_include_path(get_include_path().":../project/libraries:../php/libraries:");
-ini_set('default_charset', 'utf-8');
```
Comment (mohadesz) [inline htdocs/feedback_mri_popup.php]: require_once __DIR__ . "/../vendor/autoload.php";

File: htdocs/lost_password.php [hunk 3]
```diff
@@ -1,11 +1,20 @@
 <?php
-set_include_path(get_include_path().":../project/libraries:../php/libraries:");
-ini_set('default_charset', 'utf-8');
```
Comment (mohadesz) [inline htdocs/lost_password.php]: require_once __DIR__ . "/../vendor/autoload.php";

File: htdocs/mri/jiv/get_file.php [hunk 4]
```diff
@@ -25,7 +24,7 @@
 set_include_path(
     get_include_path() . ":../../../project/libraries:../../../php/libraries"
 );
-// Since we're sending binary data, we don't want PHP to print errors or warnings
```
Comment (mohadesz) [inline htdocs/mri/jiv/get_file.php]: require_once __DIR__ . "/../vendor/autoload.php";

File: htdocs/request_account/process_new_account.php [hunk 5]
```diff
@@ -1,135 +1,148 @@
...
- * PHP Version 5
- *
- * @category Main
- * @package  Loris
- * @author   Rathi Gnanasekaran <sekaranrathi@gmail.com>
- * @license  Loris license
- * @link     https://www.github.com/aces/Loris-Trunk/
- */
-set_include_path(get_include_path().":../../project/libraries:../../php/libraries:");
-ini_set('default_charset', 'utf-8');
```
Comment (mohadesz) [inline htdocs/request_account/process_new_account.php]: require_once __DIR__ . "/../vendor/autoload.php";

File: htdocs/survey.php [hunk 6]
```diff
@@ -69,7 +69,7 @@ function initialize()
         $this->key = $_REQUEST['key'];
```
Comment (mohadesz) [inline htdocs/survey.php]: add require_once __DIR__ . "/../vendor/autoload.php";

**Comment by rathisekaran** [comment]
I looked at it but this is a huge pull request, it may be easier to catch things if this was done one file at a time.

**Comment by driusan** [comment]
I think it's more efficient to do it all at once and get it done. If we do it one file at a time, we'll probably have numerous conflicts with other pull requests.

**Pull request merged**